andela/ah-titans
# #159053967 Users able to tag their articles

#### What does this PR do?
Creates a tag list feature that will enable users to tag their articles.

#### Description of tasks to be completed
Users should be able to create tags for the articles that they author. Only authenticated users should be able to tag articles.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans.git
```
```sh
$ git checkout ft-users-able-tag-their-articles-159053967
```
```sh
$ cd ah-titans/authors
```

> Add **.env** file and copy **example.env** content.
> Add your local database configuration settings to your **.env** file

```sh
$ mkvirtualenv -p python3 venv
```
```sh
$ source venv/bin/activate
```
```sh
$ source .env
```
```sh
$ pip install -r requirements.txt
```
```sh
$ python manage.py migrate
```
```sh
$ python manage.py runserver
```
>Now you can open your postman, create a user, login, and create an article with a tag list and it will appear as shown on the screenshot below.

<img width="980" alt="screen shot 2018-08-07 at 14 52 59" src="https://user-images.githubusercontent.com/27728464/43823618-594a575a-9af8-11e8-98da-2984feffd981.png">

>When you get tags they should appear as shown below.

<img width="976" alt="screen shot 2018-08-07 at 14 55 08" src="https://user-images.githubusercontent.com/27728464/43823691-9227e5f6-9af8-11e8-98c3-037f365da164.png">

>The tags will be displayed on the UI as shown below.

<img width="1280" alt="screen shot 2018-08-07 at 15 55 06" src="https://user-images.githubusercontent.com/27728464/43823760-ccc850f6-9af8-11e8-9c6f-d6be71fef857.png">


#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#159053954](https://www.pivotaltracker.com/story/show/159053954)
